d/d3-ease/d3-ease_rhel_8.4.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : d3-ease
+# Version       : v1.0.7
+# Source repo   : https://github.com/d3/d3-ease
+# Tested on	: UBI 8.5
```

Comment:
@vathsalashetty25 there is a discrepancy between this line and the distro mentioned in the name of build script, please check